src/root-order-basket.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { defineConfigSchema } from '@openmrs/esm-config';
+import openmrsRootDecorator from '@openmrs/react-root-decorator';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'unistore/react';
+import { orderBasketStore } from './order-basket-store';
+import OrderBasket from './order-basket/order-basket.component';
+import { switchTo } from '@openmrs/esm-extensions';
+import { useCurrentPatient } from '@openmrs/esm-api';
+
+defineConfigSchema('@openmrs/esm-drugorder-app', {});
+
+export interface RootOrderBasketProps {
+  patientUuid?: string;
+  closeWorkspace?: () => void;
+}
+
+function RootOrderBasket({ patientUuid, closeWorkspace: closeWorkspace }: RootOrderBasketProps) {
+  const [, , fallbackPatientUuid] = useCurrentPatient();
+  patientUuid = patientUuid ?? fallbackPatientUuid;
```

Comment:
@FlorianRappl I know that you said that this shouldn't be in the root, but to me it feels the most natural in this location. If it remains here, all the fallbacks are in one location and we don't have to deal with `undefined`s in lower layers.

Reply:
Sounds good to me - thanks for providing the insights!